site/docs/integrations/docker-extension.md

```diff
@@ -2,4 +2,115 @@
+
+   ![Livecycle Docker Extension](docker-ext-4.png)
+
+4. **Accessing the shared URL**
+
+   Once you have shared the URL with your team members, they can access the URL in their browser. They will be able to see the application running in their browser.
+
+   ![Livecycle Docker Extension](docker-ext-5.png)
+
+5. **Accessing Livecycle dashboard**
+
+   You can also access the Livecycle dashboard to see the logs and debug your application. Click on the "Open Link" button to open the Livecycle dashboard. On the dashboard, you can see all the running applications and services. You can also see the public and private URLs for each service.
+
+   ![Livecycle Docker Extension](docker-ext-6.png)
+
+6. **Debugging, inspecting, and logging**
+
+   Once you have opened the Livecycle dashboard, you can see all the environments/apps that are running. Click on the name of the environment for which you want to see the logs, terminal, etc. You can view the logs, terminal, and container inspection for each service.
+
+   ![Livecycle Docker Extension](docker-ext-7.png)
+
+That's it! You have successfully installed the Livecycle Docker Extension and shared your local development environment with your team.
+
+## FAQ
+
+<details>
+  <summary>How Livecycle Docker Extension is different from ngrok?</summary>
+  <p>
+    The main difference is the collaboration aspect. Livecycle allows you to share your local development environment with your team, so you can get feedback much earlier in the development workflow.
+  </p>
+</details>
+
+
+<details>
+  <summary>Which frameworks and languages does Livecycle support?</summary>
+  <p>
+    Livecycle is agnostic to specific language or framework. It works with any language or framework that can be run in a Docker container.
+  </p>
+</details>
+
+<details>
+  <summary>Is Livecycle Docker Extension secure?</summary>
+  <p>
+    Yes, Livecycle Docker Extension is secure. It uses a private tunnel to expose your local development environment to the internet. You can also add access controls to restrict access to your environment. In addition, Livecycle Docker Extension supports HTTPS and SSL termination.
+  </p>
+</details>
+
+<details>
+  <summary>How do I get started with Livecycle Docker Extension?</summary>
+  <p>
+    You can get started with Livecycle Docker Extension by following the steps in the <a href="#getting-started">Getting Started</a> section.
+  </p>
+</details>
+
+<details>
+  <summary>What makes Livecycle Docker Extension different from other tools?</summary>
+  <p>
+    Support provisioning to your cloud provider or k8s cluster -> In that context we are not bound to abilities like tunnel or ngrok as we allow the environment to be available and accessible regardless of your local setup. We also support your CI pipeline- showing that it can be added to your GH actions for example while also add comments to your GH.
```

Comment:
This explanation needs some clarification - you can't really do those things with just the extension.

```suggestion
    Support provisioning to your cloud provider or k8s cluster -> In that context we are not bound to abilities like tunnel or ngrok as we allow the environment to be available and accessible regardless of your local setup. We also support your CI pipeline- showing that it can be added to your GH actions for example while also add comments to your GH.
```

Reply:
Hey, thanks

Can you tell me what should I write instead of that?